package.json

```diff
@@ -14,7 +14,7 @@
   }
 , "dependencies":
   { "strscan": ">=1.0.1"
-  , "coffee-script": ">=1.2.0"
+  , "coffeescript": "^1.12.7"
```

Comment:
Given that this matches what we've [been using](https://github.com/mavenlink/mavenlink/blob/df7d8ab36edcbc7fae3190f97ff9aa16554530d4/yarn.lock#L2405), this should be good!

Reply:
Thanks Alan. Re

> An additional check would be to point to this branch and observe that the test suite is green before merging this in.

I think that's a great idea Alan, and we'll need a PR to update our reference back anyway, so doing that should give us more confidence: https://github.com/mavenlink/mavenlink/pull/16795

I'm assuming this will go like:

1. Ensure that bigmaven goes green
1. Merge this
1. Point back to `eco#master` from bigmaven, re-run `yarn install` and check in the update
1. Ensure that again goes green
1. Merge that in to bigmaven

Reply:
Yep, just like if this were a `mavenlink-js` change!